### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by clementbiron**
Application and adaptation of the [improvements made](https://github.com/OpenTermsArchive/template-declarations/pull/26#issuecomment-1470460781) on the readme template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,35 @@
 
 These services declarations are added and maintained by volunteer contributors, with occasional reviews from the Open Terms Archive core team. Most of these declarations were historically imported from TOSBack.org.
 
-These service declarations are consumed by the [Open Terms Archive engine](https://github.com/OpenTermsArchive/engine). See the Open Terms Archive [contribution guidelines](https://github.com/OpenTermsArchive/engine/blob/main/docs/doc-contributing-documents.md) to learn how to add and update services or documents.
+These service declarations are consumed by the [Open Terms Archive engine](https://github.com/OpenTermsArchive/engine).
```
Comment (MattiSG) [inline README.md:5]:
```suggestion
The terms listed in this collection are tracked by the [Open Terms Archive engine](https://github.com/OpenTermsArchive/engine). 
```

**Review by MattiSG — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -2,20 +2,35 @@
...
 
-The documents declared in this repository are:
+The terms tracked by this repository are those of services that operate:
 
+- In any industry.
 - In any jurisdiction.
 - In the English language.
 
-Please note that due to its specific history of being the first instance of Open Terms Archive, before a clear architectural choice for decentralisation was taken, and being mostly based on non-curated imports of an existing fonds, this scope is unusually large and inconsistently enforced. We strongly encourage contributors to focus on [topical instances](https://github.com/OpenTermsArchive/engine/blob/main/README.md#instances) and use this instance only as a last resort.
+Please note that due to its specific history of being the first instance of Open Terms Archive, before a clear architectural choice for decentralisation was taken, and being mostly based on non-curated imports of an existing fonds, this scope is unusually large and inconsistently enforced. We strongly encourage contributors to focus on [topical instances](https://docs.opentermsarchive.org/#instances) and use this instance only as a last resort.
```
Comment (MattiSG) [inline README.md:19]: I feel like the link should be more a link to the list of instances than to the documentation of what instances are, WDYT? 🙂 

```suggestion
Please note that due to its specific history of being the first instance of Open Terms Archive, before a clear architectural choice for decentralisation was taken, and being mostly based on non-curated imports of an existing fonds, this scope is unusually large and inconsistently enforced. We strongly encourage contributors to focus on [topical collections](https://opentermsarchive.org/#collections) and use this collection only as a last resort.
```

**Review by MattiSG — APPROVED** [review]
Thanks @clementbiron!

File: README.md (hunk 2 above)
Comment (clementbiron) [reply README.md]: Totally agree

**Pull request merged**